packages/plugin-react-refresh/README.md

```diff
@@ -26,6 +26,16 @@ export default {
 }
 ```
 
+## Specifying Additional Babel Plugins
+
+```js
+export default {
+  plugins: [reactRefresh({
+    plugins: ['@emotion/babel-plugin']
```

Comment:
```suggestion
    babelPlugins: ['@emotion/babel-plugin']
```

Should we be more explicit here?

Reply:
I think `plugins` is ok, like `parserPlugins`. The users should be aware the `react-refresh` is construct on babel.

```
reactRefresh({
    parserPlugins: [
      'classProperties',
      'classPrivateProperties'
    ]
  })
```